autonomy/cli/core.py

```diff
@@ -23,6 +23,9 @@
 from aea.cli.core import cli
 
 import autonomy
+
+# trigger patching of 'aea scaffold' command
+import autonomy.cli.scaffold_fsm  # noqa
```

Comment:
why not take the explicit approach below with `cli.add_command()`?

Reply:
just because I wanted to add a new subcommand to `aea scaffold`. The command group `scaffold` is already defined and added in `aea.cli`, so we cannot use the approach below (afaics they are all new commands, and do side-effect on the AEA CLI function ` from aea.cli.core import cli`).

However, we can definitely add a completely new command, not based on `aea scaffold`. Let me know what you prefer

Reply:
Ok I see, no this works then!